dashboard/app/controllers/home_controller.rb

```diff
@@ -34,6 +34,23 @@ def health_check
 
   GALLERY_PER_PAGE = 5
   def index
+    if request.cookies['pm'] == 'new_header'
+      redirect_to '/home'
+    else
+      init_homepage
+    end
+  end
+
+  def home
+    if request.cookies['pm'] != 'new_header'
+      redirect_to '/'
+    else
+      init_homepage
+      render 'home/index'
+    end
+  end
+
+  def init_homepage
```

Comment:
Not quite sure what you mean here, but certainly the assumption is that the `index` function is implicitly rendering `home/index` as well.